dnf-behave-tests/features/alias-command.feature

```diff
@@ -32,7 +36,57 @@ Scenario: Delete alias
    When I execute dnf with args "alias list"
    Then the exit code is 0
    And stdout does not contain "Alias inthrone"
```

Comment:
Or stdout is
"""
No aliases defined.
"""

Reply:
I added commit that changes all stdout checks in this feature file.